README.md

```diff
@@ -1,95 +1,251 @@
 # pymmcore-gui
 
-This is a stub repo for discussing a unified effort towards a GUI application
-for [`pymmcore-plus`](https://github.com/pymmcore-plus/pymmcore-plus) &
-[`pymmcore-widgets`](https://github.com/pymmcore-plus/pymmcore-widgets)
+*Name pending – this is a working title for the project*
+
+**pymmcore-gui** is a Graphical User Interface application for controlling
+microscopes via the Micro-Manager core – completely in Python. It unifies the
+capabilities of several libraries in the pymmcore-plus ecosystem under one
+application. The goal is to provide a pure-Python replacement for the
+traditional Java-based [Micro-Manager
+GUI](https://github.com/micro-manager/micro-manager), leveraging modern Python
```

Comment:
Maybe worth calling this MMStudio (and its plugins): `the traditional Java-based Micro-Manager GUI, also known as [MMStudio, and its plugins](https://github.com/micro-manager/micro-manager), ...`